### PR TITLE
Issue 11

### PR DESCRIPTION
**Pull request by ThomasRuby**
See #11, should be ok now.

### CODE REVIEW

**Review by nkrusch — APPROVED** [review]

**Comment by nkrusch** [comment]
Looks good to me!

**Comment by nkrusch** [comment]
After merging this `Analysis.py` (old) gives different results, we should not use it.

File: c_files/infinite/exponent.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ */
+
+int main(){
+
+    {
+        if (n % 2 == 1)
+            r *= p;
+        p *= p;
+        n /= 2;
+    }
```
Comment (nkrusch) [inline c_files/infinite/exponent.c:13]: why braces around here? is that valid?

File: c_files/other/braces_issues.c [hunk 2]
```diff
@@ -0,0 +1,7 @@
+int main() {
+    int x;
+    int y;
+    {
+    if(x > y){x = y;}
+}
```
Comment (nkrusch) [inline c_files/other/braces_issues.c:6]: (same question about the outer braces)

File: c_files/infinite/exponent.c (hunk 1 above)
Comment (nkrusch) [reply c_files/infinite/exponent.c:13]: I have resolved this myself and it seems to be OK, closing my own comment.

**Review by nkrusch — APPROVED** [review]

File: c_files/other/braces_issues.c (hunk 2 above)
Comment (aubertc) [reply c_files/other/braces_issues.c:6]: Cf. https://github.com/seiller/pymwp/issues/25 
As far as I know this is valid C code.

**Comment by nkrusch** [comment]
I will update the readmes to reflect path changes in the examples

File: c_files/other/braces_issues.c (hunk 2 above)
Comment (nkrusch) [reply c_files/other/braces_issues.c:6]: CompCert says it is OK so must be :)

**Comment by ThomasRuby** [comment]
Thanks @nkrusch :+1:

**Pull request merged**